{"name": "Тест невозможности авторизации зарегистрированного курьера при вводе некорректных данных (статус ответа 404)", "status": "passed", "steps": [{"name": "Авторизация курьера", "status": "passed", "parameters": [{"name": "login", "value": "'zyqxettrok'"}, {"name": "password", "value": "'incorrect_data'"}], "start": 1734893674611, "stop": 1734893674885}], "parameters": [{"name": "field_name", "value": "'password'"}], "start": 1734893674611, "stop": 1734893674885, "uuid": "014f2381-0af3-4b74-98f8-126688cb32e2", "historyId": "78bd892d92288947a085d1996bcd5f2b", "testCaseId": "62de0a1455e4348f76fb13991ea19ed5", "fullName": "tests.couriers.test_login_courier.TestLoginCourier#test_login_courier_with_incorrect_login_or_password", "labels": [{"name": "parentSuite", "value": "tests.couriers"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-TPCCV60"}, {"name": "thread", "value": "13868-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.couriers.test_login_courier"}]}